AISKU/src/Initialization.ts

```diff
@@ -19,16 +24,20 @@ export interface IApplicationInsights extends IAppInsights, IDependenciesPlugin,
     appInsights: ApplicationInsights;
 };
 
+/**
+ * Application Insights API
+ * @class Initialization
+ * @implements {IApplicationInsights}
+ */
 export class Initialization implements IApplicationInsights {
```

Comment:
One thing we are missing here is making channels available or just the flush api. its a common requirement.

Reply:
Internally, it will call getchannelcontrols().foreach(c => c.flush)

Reply:
got it. will add for next beta version